astropy/tests/helper.py

```diff
@@ -232,7 +232,7 @@ def treat_deprecations_as_exceptions():
     # on. See https://github.com/astropy/astropy/pull/5513.
     for module in list(sys.modules.values()):
         # We don't want to deal with six.MovedModules, only "real"
-        # modules.
+        # modules. FIXME: we no more use six, this should be useless ?
```

Comment:
I see that the FIXME comment was removed, but nothing was done about a potentially "useless" code that follows. 🤔

Reply:
I was not sure what to do with this, but now I think that we can keep the check (`isinstance(module, types.ModuleType)`) and just the comment about six. It was added for six but this is just checking that the object is a real module.